modules/core/src/lighting/point-light.js

```diff
@@ -0,0 +1,8 @@
+import {PointLight as BasePointLight} from 'luma.gl';
+
+export default class PointLight extends BasePointLight {
+  constructor(props) {
+    super(props);
+    this.coordinateSystem = props.coordinateSystem;
```

Comment:
what is the purpose of forwarding `props.coordinateSystem` to `this`?

Or is this better to be part of luma.gl?

Reply:
Fine for now. but we should add a note to the audit log document to make sure we come back to this.

My concern is that it can be confusing for users that our two closely related libraries both export classes with exactly the same name.

(We could call the deck.gl versions `DeckPointLight` etc, but it is probably too awkward. Hopefully we can brainstorm up better solutions...  perhaps we could extend the luma.gl class to save all props it is given etc).

Reply:
Give `coordinateSystem` a default value? e.g. `COORDINATE_SYSTEM.LNGLAT` if `viewport.isGeospatial`, otherwise `COORDINATE_SYSTEM.IDENTITY`.